### PR TITLE
update predict code

### PR DESCRIPTION
**Pull request by sayatmimar**
This PR updated the prediction code to run in DSA.

### CODE REVIEW

File: histomicstk/segmentationschool/Codes/IterativePredict_1X.py [hunk 1]
```diff
@@ -205,8 +202,11 @@ def predict(args):
             print(basename)
             # print(extname)
             # try:
-            slide=openslide.TiffSlide(wsi)
-            print(wsi,'here/s the silde')
+            try:
+                slide=openslide.TiffSlide(wsi)
```
Comment (anindya-paul) [inline histomicstk/segmentationschool/Codes/IterativePredict_1X.py:206]: You can do openslide.OpenSlide(wsi') as tiffslide is a drop-in replacement for openslide and tiffslide has been imported as openslide.

File: histomicstk/segmentationschool/segmentation_school.py [hunk 2]
```diff
@@ -273,7 +273,7 @@ def savetime(args, starttime):
 
 
     ### Params for optimizing wsi mask cleanup ###
-    parser.add_argument('--min_size', dest='min_size', default=[30,30,24000,24000,10,10] ,type=int,
+    parser.add_argument('--min_size', dest='min_size', default=[30,30,30,30,30,30] ,type=int,
```
Comment (anindya-paul) [inline histomicstk/segmentationschool/segmentation_school.py:276]: @sayatmimar: can you elaborate what this default to --min_size mean? Why did it need to change?

File: histomicstk/segmentationschool/Codes/IterativePredict_1X.py
Comment (anindya-paul) [inline histomicstk/segmentationschool/Codes/IterativePredict_1X.py:1]: Overall, there is minimal change. The major change is removing the zoom each patch to 4x before panoptic segmentation. I agree, let's deploy this to production but point to note that we need to do more QC/QA now internally on PAS WSIs before we open up to external folks for their QC/QA.

Also, @sayatmimar, can we do some code clean up? There is lots of commented lines, now we have git histories so let's remove them before prod deployment. Good job removing all those print statements.

**Review by anindya-paul — COMMENTED** [review]
Added my minor comments for clarifications, overall good to go to DSA.

File: histomicstk/segmentationschool/Codes/IterativePredict_1X.py
Comment (sayatmimar) [reply histomicstk/segmentationschool/Codes/IterativePredict_1X.py:1]: @anindya-paul I agree, there's so much commented lines unnecessary code/functions. I did some cleanup in the predict file, we should clean the code in other files as well, as we work on them

File: histomicstk/segmentationschool/segmentation_school.py (hunk 2 above)
Comment (sayatmimar) [reply histomicstk/segmentationschool/segmentation_school.py:276]: That's a good question, in this version of the code this min size parameter was changed for some reason. It seems that's the min size of a contour,  and contour are below this value is ignored (see [this line](https://github.com/SarderLab/Multi-Compartment-Segmentation/blob/fe260c2857471fd8ecd95f15c070b85f05f59f8d/histomicstk/segmentationschool/Codes/IterativePredict_1X.py#L469)). I did not write the code so I don't have a good answer. Maybe @njlucare knows? Or we can ask Brandon.

Comment (anindya-paul) [reply histomicstk/segmentationschool/segmentation_school.py:276]: Hmm I understand the line you pointed still I don't understand why the number of list elements to be 6 also what prompted changing the value from 24000 to 30, seems odd. @njlucare can you help?

File: histomicstk/segmentationschool/Codes/IterativePredict_1X.py
Comment (anindya-paul) [reply histomicstk/segmentationschool/Codes/IterativePredict_1X.py:1]: Great, thanks! I'm going to approve this PR now.

**Review by anindya-paul — APPROVED** [review]

**Pull request merged**